tests/fuzzer/fuzzer.py

```diff
@@ -35,6 +35,7 @@ class ErrorType(str, Enum):
     SCHEDULE = "SCHEDULE"
     RESULT = "RESULT"
     STDCM = "STDCM"
+    INFRA = "INFRA"
```

Comment:
This enum is used when reproducing tests with the same random elements. There's no randomness on infra loading here, I don't think it's really relevant